qcustomplot-add-cmake.patch

```diff
@@ -0,0 +1,145 @@
+--- /dev/null
```

Comment:
QCustomPlot is distributed without any build system manifest (there is a sample qmake file in additional "examples" archive). This is intentional and QCustomPlot upstream is not interested in the CMake file.